tensorflow/compiler/tests/slice_ops_test.py

```diff
@@ -0,0 +1,136 @@
+# Copyright 2017 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Tests for slicing."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import numpy as np
+
+from tensorflow.compiler.tests.xla_test import XLATestCase
+from tensorflow.python.framework import dtypes
+from tensorflow.python.ops import array_ops
+from tensorflow.python.platform import googletest
+
+class SliceTest(XLATestCase):
+
+  def test1D(self):
+    with self.test_session():
+      i = array_ops.placeholder(dtypes.float32, shape=[10])
```

Comment:
Since none of these test cases are specific to a particular type, the test coverage would be better if you wrapped these tests in a loop of the form:
```
for dtype in self.numeric_types:
  ...
```
and used `dtype` as the dtype where applicable.